config/locales/en.yml

```diff
@@ -2072,129 +2049,40 @@ en:
-          Spanish National Geographic Institute (%{ign_link}) and
-          National Cartographic System (%{scne_link})
-          licensed for reuse under %{cc_by_link}.
-        contributors_es_spain: Spain
-        contributors_es_ign: IGN
-        contributors_es_ign_url: https://www.ign.es/
-        contributors_es_scne: SCNE
-        contributors_es_scne_url: https://www.scne.es/
-        contributors_es_cc_by: CC BY 4.0
-        contributors_es_cc_by_url: https://creativecommons.org/licenses/by/4.0/
-        contributors_za_credit_html: |
-          %{south_africa}: Contains data sourced from %{ngi_link}, State copyright reserved.
-        contributors_za_south_africa: South Africa
-        contributors_za_ngi: "Chief Directorate: National Geo-Spatial Information"
-        contributors_za_ngi_url: https://ngi.dalrrd.gov.za/
-        contributors_gb_credit_html: |
-          %{united_kingdom}: Contains Ordnance
-          Survey data &copy; Crown copyright and database right
-          2010-2023.
-        contributors_gb_united_kingdom: United Kingdom
-        contributors_2_html: |
-          For further details of these, and other sources that have been used
-          to help improve OpenStreetMap, please see the %{contributors_page_link} on the OpenStreetMap Wiki.
-        contributors_2_contributors_page: Contributors page
-        contributors_2_contributors_page_url: https://wiki.openstreetmap.org/wiki/Contributors
+          OpenHistoricalMap is the collaborative effort of thousands of individuals and organizations, who in turn draw from a large body of scholarship, journalism, and recordkeeping.
+          Our most substantial sources and contributors include:
+        contributors_anr_communes_html: |
+          Data created by the %{anr_communes_link} project in collaboration with %{campop_link}.
+        contributors_anr_communes_project: ANR COMMUNES
+        contributors_anr_communes_campop: The Cambridge Group for the History of Population and Social Structures
+        contributors_anr_communes_campop_url: https://www.campop.geog.cam.ac.uk/research/projects/internationaloccupations/enchpopgos/france/
+        contributors_newberry_ahcb_html: |
+          Data from the %{ahcb_link}, a project of the William M. Scholl Center for American History and Culture at %{newberry_link}, which is available for reuse without restriction.
+        contributors_newberry: The Newberry Library
+        contributors_newberry_ahcb: Atlas of Historical County Boundaries
+        # https://wiki.openstreetmap.org/wiki/Legal_FAQ/CC-BY-SA_Archive#I_would_like_to_use_OpenStreetMap_maps._How_should_I_credit_you?
+        contributors_osm_html: |
+          Data about coastlines, bodies of water, and streams © %{osm_link}, %{cc_by_sa_link}.
+        contributors_osm: OpenStreetMap contributors
+        contributors_osm_cc_by_sa: CC BY-SA 2.0
+        contributors_osm_cc_by_sa_url: https://creativecommons.org/licenses/by-sa/2.0/
```

Comment:
OSM is ODbL-licensed nowadays, but the reason we’re crediting OSM here is that the CC&nbsp;BY-SA–licensed [coastline import](https://wiki.openstreetmap.org/wiki/OpenHistoricalMap/Imports/OpenStreetMap_water) makes up so much of the database. The ODbL-licensed content is much more minor by comparison.